src/services/sharedFolder.js

```diff
@@ -1,21 +1,25 @@
-var utils = require('../utils')
-var isbinaryfile = require('isbinaryfile')
-var fs = require('fs-extra')
-var chokidar = require('chokidar')
+const utils = require('../utils');
+const isbinaryfile = require('isbinaryfile');
+const fs = require('fs-extra');
+const chokidar = require('chokidar');
 
 module.exports = {
```

Comment:
I would recommend at some point to switch to a "class based", or "function with closures" version of this object to make it more readable.

Reply:
This is a comment. I don't expect you to do it for this PR

Reply:
Okay, will leave this here for the future, sharedFolder was part of remixd before and all the logic is the same, I've just updated var to let or const. Class based sounds good so that we are consistent across whole remixd.

Reply:
Resolving this as this won't be part of this PR.